src/Altinn.Notifications/Migration/v0.02/01-setup-functions.sql

```diff
@@ -0,0 +1,42 @@
+-- Function: notifications.get_unsenttargets
+CREATE OR REPLACE FUNCTION notifications.get_unsenttargets()
+    RETURNS TABLE (
+        id bigint,
+        notificationid bigint,
+        channeltype character varying,
+        "address" character varying,
+        "sent" timestamptz)
+    LANGUAGE 'plpgsql'
+
+AS $BODY$
+BEGIN
+return query
+	SELECT t.id, t.notificationid, t.channeltype, t."address", t."sent"
+    WHERE t."sent" IS NULL
+	FROM notifications.targets t;
```

Comment:
We usually place the FROM clause before the WHERE clause. Curious that this even works.